src/AaveV3AvaxRatesUpdates_20230331/AaveV3AvaxRatesUpdates_20230331.t.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import 'forge-std/Test.sol';
+
+import {IPoolConfigurator, ConfiguratorInputTypes} from 'aave-address-book/AaveV3.sol';
+import {AaveV3Avalanche} from 'aave-address-book/AaveAddressBook.sol';
+import {ProtocolV3TestBase,ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
+import {AaveV3AvalancheAssets} from 'aave-address-book/AaveV3Avalanche.sol';
+import {AaveV3AvaRatesUpdatesSteward_20230331} from './AaveV3AvaxRatesUpdatesSteward_20230331.sol';
+
+contract AaveV3AvaCapsByGuardian is ProtocolV3TestBase, TestWithExecutor {
+    using stdStorage for StdStorage;
+
+    address public constant GUARDIAN_AVALANCHE =
+      0xa35b76E4935449E33C56aB24b23fcd3246f13470;
+
+    function setUp() public {
+      vm.createSelectFork(vm.rpcUrl('avalanche'), 28120569);
+      _selectPayloadExecutor(GUARDIAN_AVALANCHE);
+    }
+
+    function testUpdatesRatesAndReserveFactors() public {
```

Comment:
Not a big deal, but I would validate the Interest rate strategy as well :)

Reply:
agreed, was following the example from the other rates updates payloads and tests - it seems that the new reports feature being built out will make it much easier to validate the actual rate strategy params rather than relying on the correctness of the update engine